graphics/dcmtk/Portfile

```diff
@@ -59,18 +59,6 @@ configure.args-append   -DDCMTK_WITH_TIFF=OFF \
                         -DDCMTK_WITH_WRAP=ON \
                         -DDCMTK_ENABLE_CXX11=OFF
 
-patch {
-    # DCMTK by default installs headers with PACKAGE_VERSION and friends
-    # defined, rather than some more unique name. We'll just change that for
-    # them...
-    set replstring \
```

Comment:
Guess I'll push for the change upstream and then for openimageio to update to match.